scripts/validate_billing_e2e_sessions.py

```diff
@@ -0,0 +1,35 @@
+import os
+import urllib.request
+import json
+import time
+import datetime
+
+def dateToMillis(date):
+    return int(time.mktime(date.timetuple()) * 1000)
+
+token = os.environ["MAPBOX_BILLING_STATISTICS_ACCESS_TOKEN"]
+
+periodEndTime = datetime.datetime.now()
+periodEndTimeMillis = dateToMillis(periodEndTime)
+
+periodStartTime = periodEndTime - datetime.timedelta(days=1, hours = 1)
+periodStartTimeMillis = dateToMillis(datetime.datetime.now() - datetime.timedelta(days=1, hours = 1))
```

Comment:
Why do we take 25 hours?

Reply:
I've fixed it to 24 hours.

In this script we verify billing session made a day before because billing sessions are not available in database immediately, usually it takes about 5-6 hours for a session to appear in billing statistics. And actually even 24 hours time window is quite a big, but it doesn't matter, we verify only number of sessions. We specify 24 hours time window to avoid flakiness related to database delays.